.github/workflows/scripts/only_latest_easystacks.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+EESSI_VERSION=${EESSI_VERSION:-"2023.06"}
```

Comment:
If you source `init/eessi_defaults` you don't have to change the version here later.

Reply:
I wanted this to be configurable in CI via an setting the envvar

Reply:
The value here is actually set in the CI configuration, this is only here as a fall back in case someone explicitly calls this script for some reason

Reply:
Benefit of sourcing `init/eessi_defaults` is that one has to change the default in a single place only. In CI, one can set `EESSI_VERSION_OVERRIDE` before sourcing the defaults script.

Anyhow, I'll approve the PR.